logback-access/src/main/java/ch/qos/logback/access/spi/AccessEvent.java

```diff
@@ -500,9 +509,14 @@ public String getRequestContent() {
                         buf.append("");
                     }
                 }
-            } catch (Exception e) {
-                // FIXME Why is try/catch required?
-                e.printStackTrace();
+            } catch (Throwable t) {
```

Comment:
Ditto here.

The Servlet Spec doesn't specify the exceptions that can happen here.
It can be something as simple as an IOException (which is a checked exception) that has to be wrapped in something else.
It can be a parsing issue, it can be a Socket issue, it can be charset issue, etc...

Unlike a normal webapp and how it handles failure, a RequestLog cannot skip its logging.
Ideally, a RequestLog should never call a method that changes the Request or Response state.